src/main/scala/org/coroutines/Coroutine.scala

```diff
@@ -166,12 +201,29 @@ object Coroutine {
 
     final def hasException: Boolean = isCompleted && $exception != null
 
+    /** Returns whether or not the coroutine has more instances to go through.
+      * This is true if there are either more yield statements or if the
+      * coroutine has not yet returned its result.
+      * 
+      * @return `true` if `resume` can be called without an exception being
+      *         thrown, `false` otherwise.
+      */
     final def isLive: Boolean = $costackptr > 0
 
+    /** Returns whether or not the coroutine has completed. See the
+      * documentation for `isLive`.
+      * 
+      * @return `!isLive`.
+      */
     final def isCompleted: Boolean = !isLive
 
+    /** Returns a string representation of the coroutine's status.
+      * 
+      * @return A string containing the values of `\$costackptr` and `isLive`.
```

Comment:
Make sure you don't refer to internal implementation details from the doc.

`A string describing the coroutine state.`